src/test/java/testresults/GetTestResults.java

```diff
@@ -359,102 +359,5 @@ public void testResultsWithTestVersionAutomaticallySetSame() {
         testResultsSteps.statusCodeShouldBe(200);
         testResultsSteps.valueForFieldInPathShouldBe("[0].testVersion","archived");
         testResultsSteps.valueForFieldInPathShouldBe("[0].testHistory[0].testVersion","archived");
-
```

Comment:
Why were these tests deleted?

Reply:
these tests are moved to PutTestResults class